ui/src/components/Investor/InvestorTrade.tsx

```diff
@@ -116,7 +118,10 @@ const InvestorTrade: React.FC<Props> = ({ deposits, sideNav, onLogout }) => {
                     <div className='order-input'>
                         <OrderForm
                             allowedToOrder={
-                                !!ccpCustomerContracts.find(ccp => ccp.contractData.ccpCustomer === investor)
+                                !!ccpCustomer
+                            }
+                            inGoodStanding={
+                                ccpCustomer ? ccpCustomer.contractData.inGoodStanding : false
```

Comment:
You could do 

`!!ccpCustomer?.contractData?.inGoodStanding` 

(and have it fit on one line)